routes/discordactions.js

```diff
@@ -39,13 +39,21 @@ router.get("/invite", authenticate, getUserDiscordInvite);
 router.post("/invite", authenticate, checkCanGenerateDiscordLink, generateInviteForUser);
 router.delete("/roles", authenticate, checkIsVerifiedDiscord, deleteRole);
 router.get("/roles", authenticate, checkIsVerifiedDiscord, getGroupsRoleId);
+// TODO deprecate the below API, for incorrect naming
 router.patch(
   "/avatar/verify/:id",
   authenticate,
   authorizeRoles([SUPERUSER]),
   checkIsVerifiedDiscord,
   updateDiscordImageForVerification
 );
+router.patch(
+  "/avatar/update/:discordId",
```

Comment:
Is the API naming correct?

Reply:
now that you have mentioned it, I think I can come up with some better naming convention.